### PR TITLE
Add detailed info for OpenStack

### PR DESCRIPTION
**Pull request by ZhengZhenyu**
Add detailed info for OpenStack.

### CODE REVIEW

File: _posts/cloud/2021-01-29-OpenStack.md [hunk 1]
```diff
@@ -10,8 +10,61 @@ project:
   title: "OpenStack"
   type: "Python"
   ci_url: "https://zuul.opendev.org/t/openstack/status"
-  url: "https://zuul.opendev.org/t/openstack/status"
+  url: "OpenStack"
```
Comment (martin-g) [inline _posts/cloud/2021-01-29-OpenStack.md:13]: is this intentional ?
It is not really a URL anymore.

File: _posts/cloud/2021-01-29-OpenStack.md [hunk 2]
```diff
@@ -10,8 +10,61 @@ project:
...
   logo: "/assets/images/cloud/openstack/logo.png"
+  overview: "OpenStack is a cloud operating system that controls large pools of compute, storage, and networking resources throughout a datacenter, all managed and provisioned through APIs with common authentication mechanisms.Beyond standard infrastructure-as-a-service functionality, additional components provide orchestration, fault management and service management amongst other services to ensure high availability of user applications."
```
Comment (martin-g) [inline _posts/cloud/2021-01-29-OpenStack.md:15]: Missing space after the dot at `.Beyond ...`

File: _posts/cloud/2021-01-29-OpenStack.md (hunk 1 above)
Comment (martin-g) [reply _posts/cloud/2021-01-29-OpenStack.md:13]: Maybe use https://www.openstack.org ?

Comment (ZhengZhenyu) [reply _posts/cloud/2021-01-29-OpenStack.md:13]: Yes, OpenStack will join others and become the url like this:
https://kunpengcompute.github.io/arm-landscape/OpenStack/

File: _posts/cloud/2021-01-29-OpenStack.md [hunk 3]
```diff
@@ -10,8 +10,61 @@ project:
...
+  - release:
+    version: "Victoria"
+    url: "https://www.openstack.org/software/victoria/"
+  - release:
+    version: "Wallaby"
+    url: "https://www.openstack.org/software/wallaby/"
+
+project_ci:
+    - CI:
+      title: "Hadoop Upstream Arm CI"
```
Comment (martin-g) [inline _posts/cloud/2021-01-29-OpenStack.md:42]: Hadoop ?!

Comment (ZhengZhenyu) [reply _posts/cloud/2021-01-29-OpenStack.md:42]: Will fix in next pr

**Pull request merged**